megamek/src/megamek/server/Server.java

```diff
@@ -9956,6 +10022,64 @@ else if (bldgExited.equals(bldgEntered)
             }
         }
     }
+    
+    /**
+     * Updates the position of any towed trailers.
+     *
+     * @param tractor    The Entity that is moving
+     * @param trailer    The current trailer being updated
+     */
+    private void processTrailerMovement(Entity tractor, Entity trailer, ArrayList<Coords> trainPath) {
```

Comment:
`List<Coords> trainPath`

Reply:
Fixed this and other references to trainPath.